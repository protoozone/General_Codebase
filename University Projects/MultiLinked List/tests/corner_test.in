REMOVE 0
DELETE 0 0
INSERT 0 0
REMOVE 1
DELETE 0 1
INSERT 0 1
REMOVE -1
DELETE 1 0
INSERT 1 0
REMOVE  0
DELETE  0  0
INSERT  0  0
 REMOVE 0
DELETE 0  0
INSERT 0  0
l REMOVE 0
 DELETE 0 0
 INSERT 0 0
NEW  0
VIEW 0
VIEW 1
VIEW -1
VIEW ALL
VIEW-NESTED 0
VIEW-NESTED 1
VIEW-NESTED -1
NEW -1
INPUT 21
ADD  0
HELP
EXIT