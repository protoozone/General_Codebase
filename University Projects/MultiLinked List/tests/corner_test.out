INVALID COMMAND: REMOVE
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: REMOVE
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: REMOVE
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: INPUT
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: INPUT
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: DELETE
INVALID COMMAND: INSERT
INVALID COMMAND: INPUT
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
INVALID COMMAND: VIEW
Number of lists: 0
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: VIEW-NESTED
INVALID COMMAND: NEW
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: INPUT